apps/web/composables/useNotification/__tests__/useNotification.spec.ts

```diff
@@ -0,0 +1,174 @@
+import { useNotification } from '../useNotification';
+
+describe('useNotification', () => {
+
+    afterEach(() => {
+        const { clear } = useNotification();
+        clear();
+    });
+
+    it('should return the correct initial state', () => {
+        const { data, send } = useNotification();
+
+        expect(data.value).toEqual([]);
+        expect(send).toBeInstanceOf(Function);
+    });
+
+    it('should add a notification', () => {
+        const { data, send } = useNotification();
+
+        send({
+            message: 'Test alert error with a longer message',
+            type: 'negative',
+            persist: true,
+            action: {
+                text: 'action',
+                onClick: () => {},
+            },
+        });
+
+        expect(data.value.length).toBe(1);
+    });
+
+    it('should remove a notification', () => {
+        const { data, send } = useNotification();
+
+        send({
+            message: 'Test alert error with a longer message',
+            type: 'negative',
+            action: {
+                text: 'action',
+                onClick: () => {},
+            },
+        });
+
+        expect(data.value.length).toBe(1);
+
+        if (data.value[0].dismiss) {
+            data.value[0].dismiss();
+        }
+
+        expect(data.value.length).toBe(0);
+    });
+
+    it('should clear all notifications', () => {
+        const { data, send, clear } = useNotification();
+
+        send({
+            message: 'Test alert error with a longer message',
+            type: 'negative',
+            action: {
+                text: 'action',
+                onClick: () => {},
+            },
+        });
+
+        expect(data.value.length).toBe(1);
+
+        clear();
+
+        expect(data.value.length).toBe(0);
+    });
+
+    it('should remove the oldest notification when the maximum number of notifications is reached', async () => {
```

Comment:
**suggestion**: The test case currently only checks that a message gets removed, not which one. Maybe `i` could be part of the message and have an assertion to that effect.

Reply:
good idea, done 👍